Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3541,15 +3541,7 @@ - (void)updateGroupModelTo:(TSGroupModel *)newGroupModel successCompletion:(void
 
         groupThread.groupModel = newGroupModel;
         [groupThread saveWithTransaction:transaction];
-        message = [[TSOutgoingMessage alloc] initOutgoingMessageWithTimestamp:[NSDate ows_millisecondTimeStamp]
```

Comment:
I started working through the process of adding factory methods for interactions.